src/js/constants/NodeStatuses.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
can we combine this file with `NodeStatusDisplayText` like the file `HealthBarStates` in dcos-ui?